CHANGELOG.md

```diff
@@ -7,4 +7,23 @@ and this repository adheres to [Semantic Versioning](https://semver.org/spec/v2.
 
 ## Unreleased
 
+### Added
 
+- [pre-configured support for Smart
+  Agent](https://github.com/signalfx/splunk-otel-ruby/pull/8)
+- [support for SPLUNK_TRACE_RESPONSE_HEADER_ENABLED environment
+  variable](https://github.com/signalfx/splunk-otel-ruby/pull/38)
+  
+## v0.1.0 - 2022-07-27
```

Comment:
> MUST state version of OpenTelemetry repository built against if applicable

https://github.com/signalfx/gdi-specification/blob/89371378371723be768e90b6d1beb44fccf42c00/specification/repository.md?plain=1#L130

I know that it is about GH Releases, but often it is the same content as in the CHANGELOG.md.

Reply:
Added.